### PR TITLE
feat: check if a radar measurement is all zero

### PR DESCRIPTION
**Pull request by wvmcastro**
### What is this PR for

Add a check for all zero measurements. It can be used in particle filter for deciding whether or not to run a update step with a given radar measurement

### CODE REVIEW

File: src/Radar.hpp [hunk 1]
```diff
@@ -44,6 +44,8 @@ namespace radar_base {
         static void updateEchoes(Radar const& radar_echo,
             base::Angle const& yaw_correction,
             std::vector<uint8_t>& world_echoes);
+
+        bool allZero() const;
```
Comment (kapeps) [inline src/Radar.hpp:51]: some documentation would be nice

**Review by kapeps — APPROVED** [review]

File: test/test_Radar.cpp [hunk 2]
```diff
@@ -122,4 +122,14 @@ TEST_F(RadarTest,
                      heading_n2,
                      echo),
         std::runtime_error);
+}
+
+TEST_F(RadarTest, it_checks_if_measurement_is_zero)
```
Comment (seixasxbr) [inline test/test_Radar.cpp]: You could also test the other case

**Review by seixasxbr — APPROVED** [review]

File: test/test_Radar.cpp (hunk 2 above)
Comment (wvmcastro) [reply test/test_Radar.cpp]: If measurement is not zero? it is also checked here

Comment (wvmcastro) [reply test/test_Radar.cpp]: Changed the test name to make it more clear

**Pull request merged**